examples/appliance_device_snmp_v3_trap_destinations.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from config_loader import try_load_from_file
+
+# Set api_version to 600, default is 300 and this API is introduced since API 600.
```

Comment:
please update the [endpoint-support.md](https://github.com/HewlettPackard/python-hpOneView/blob/60b0971976edafd8c764e849e0262fc77f6eeff0/endpoints-support.md) for this resource if this will work for API600 as well

Reply:
@soodpr - I've already updated the [endpoint-support.md](https://github.com/gzecchi/python-hpOneView/blob/develop/endpoints-support.md) for those resources.
Did you mean for API 800?

Reply:
yes. sorry for the confusion, i thought you have already added API800 in endpoint-support.md. Anyways, can you please add API800 in the endpoint-support.md as you have implemented for API800.

Reply:
👍  Added in commit 686c8bc7ebbdb3445cab3d553534e5da8c858566